aea/skills/base.py

```diff
@@ -74,17 +74,17 @@ def __init__(
         self._is_active = True  # type: bool
         self._new_behaviours_queue = queue.Queue()  # type: Queue
         self._new_handlers_queue = queue.Queue()  # type: Queue
-        self._logger: Optional[Union[Logger, LoggerAdapter]] = None
+        self._logger: Optional[Logger] = None
 
     @property
-    def logger(self) -> Union[Logger, LoggerAdapter]:
+    def logger(self) -> Logger:
         """Get the logger."""
         if self._logger is None:
             return logging.getLogger("aea")
```

Comment:
Is this still needed?

Reply:
I think so because the logger might be not initialized by the user when the skill context is initialized programmatically